sql/moz-fx-data-shared-prod/account_ecosystem_derived/ecosystem_user_id_lookup_v1/metadata.yaml

```diff
@@ -14,8 +15,9 @@ labels:
   incremental: true
 scheduling:
   dag_name: bqetl_account_ecosystem
-  depends_on_past: True
-  referenced_tables: [['moz-fx-data-shared-prod', 'firefox_accounts_stable', 'account_ecosystem_v1']]
+  depends_on_past: true
+  referenced_tables: [['moz-fx-data-shared-prod', 'firefox_accounts_stable',
+                       'account_ecosystem_v1']]
```

Comment:
This might actually be the problematic file, put this one on a single line instead.

Reply:
When I put this on a single line and run it locally, I get a line-length error

Reply:
Should I use the disable line-length rule comment on top of it?

Reply:
What about adopting the style of the other files in similar to this and putting it like:

```
referenced_tables:
  - ["moz-fx-data-shared-prod", ...]
```

Reply:
No, that still causes an error. 

This way runs fine though.

`    - [
        [
          "moz-fx-data-shared-prod",
          "firefox_accounts_stable",
          "account_ecosystem_v1",
        ],
      ]`